src/databricks/labs/ucx/assessment/crawlers.py

```diff
@@ -53,6 +52,13 @@ class PipelineInfo:
     failures: str
 
 
+@dataclass
+class AzureServicePrincipalInfo:
+    application_id: str
+    secret_scope: str
+    secret_key: str
```

Comment:
```suggestion
    secret_key: str
    # fs.azure.account.oauth2.client.endpoint: "https://login.microsoftonline.com/${local.tenant_id}/oauth2/token",
    tenant_id: str
```

also add tenant_id if it's available